.github/workflows/ci.yml

```diff
@@ -87,6 +87,8 @@ jobs:
     steps:
       - name: Check out a copy of the repo
         uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

Comment:
It seems that we need to be specific about what to check out, in order for the Lighthouse status check to show up in a GitHub pull request:

![Lighthouse status check is visible in a pull request](https://user-images.githubusercontent.com/16869656/98447646-1d3f7200-20ec-11eb-8a64-2770eba4239e.png)

I took the 2 lines verbatim from https://github.com/GoogleChrome/lighthouse-ci/blob/master/docs/getting-started.md#additional-configuration-for-github-actions-as-ci-provider.